### DISCUSSION TITLE
menu button bars opaque/transparent.

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#1705] [Q&A]
Is it possible to make the bottom and side menu bars opaque or change their color? It's hard to see them with a bright landscape or in the daytime. And some landscapes (with grass for example) make it hard to see the icons due to clutter.

Maybe there could be a simple opaque/transparent button in a menu..?

**Comment by gzotti** [comment]
F2->Extras->Use buttons background

The left buttons are considered large enough to remain without background.
